File: src/ST.Client.Mobile.Droid.Design/app/src/main/java/net/steampp/app/ui/fragments/CommunityFixFragment.java
package net.steampp.app.ui.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.lifecycle.Observer;
import androidx.lifecycle.ViewModelProvider;

import net.steampp.app.ui.databinding.FragmentCommunityFixBinding;
import net.steampp.app.ui.viewmodels.CommunityFixViewModel;

public class CommunityFixFragment extends Fragment {

    private CommunityFixViewModel viewModel;
    private FragmentCommunityFixBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        viewModel =
                new ViewModelProvider(this).get(CommunityFixViewModel.class);

        binding = FragmentCommunityFixBinding.inflate(inflater, container, false);
        View root = binding.getRoot();

        final TextView textView = binding.textView;
        viewModel.getText().observe(getViewLifecycleOwner(), new Observer<String>() {
            @Override
            public void onChanged(@Nullable String s) {
                textView.setText(s);
            }
        });
        return root;
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }
}

File: src/ST.Client.Mobile.Droid.Design/app/src/main/java/net/steampp/app/ui/fragments/CommunityFixFragment.kt
package net.steampp.app.ui.fragments

import net.steampp.app.ui.viewmodels.CommunityFixViewModel
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import androidx.lifecycle.ViewModelProvider
import android.widget.TextView
import androidx.fragment.app.Fragment
import net.steampp.app.ui.databinding.FragmentCommunityFixBinding

class CommunityFixFragment : Fragment() {
    private var viewModel: CommunityFixViewModel? = null
    private var binding: FragmentCommunityFixBinding? = null
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?, savedInstanceState: Bundle?
    ): View? {
        viewModel = ViewModelProvider(this).get(CommunityFixViewModel::class.java)
        binding = FragmentCommunityFixBinding.inflate(inflater, container, false)
        val root: View = binding!!.root
        val textView: TextView = binding!!.textView
        viewModel!!.text.observe(viewLifecycleOwner, { s -> textView.text = s })
        return root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        binding = null
    }
}
